src/videojs-http-streaming.js

```diff
@@ -168,11 +175,13 @@ const setupEmeOptions = (hlsHandler) => {
   const player = videojs.players[hlsHandler.tech_.options_.playerId];
 
   if (player.eme) {
-    player.eme.options = videojs.mergeOptions(player.eme.options, emeOptions(
+    const sourceOptions = emeOptions(
       hlsHandler.source_.keySystems,
       hlsHandler.playlists.media(),
       hlsHandler.masterPlaylistController_.mediaTypes_.AUDIO.activePlaylistLoader.media()
-    ));
+    );
+
+    player.currentSource().keySystems = sourceOptions.keySystems;
```

Comment:
This throws an error if `hlsHandler.source_.keySystem` is `undefined`. Might be good for us to check `sourceOptions` and add a test for the case where no `keySystems` options are passed. The `emeOptions` function can also return the `keySystems` on their own now (and we can probably rename the function to `emeKeySystems`, or something like that, if we want).